R/plink_calls.R

```diff
@@ -17,10 +17,13 @@
 #'
 #' @export
 p_2_b <- function(ped_file, bed_file=ped_file, del=TRUE, plink_path=TRUE) {
-  stopifnot("ped_file needs to be a valid file" = file.exists(paste0(ped_file, ".ped")),
-            "bed_file needs to be a valid file" = TRUE,  # fix det Rasmus
-            "del needs to be either TRUE or FALSE" = class(del) == "logical",
-            "plink_path needs to be a valid path to plink" = (plink_path == TRUE || file.exists(paste0(plink_path, "/plink.exe"))))
+  stopifnot("ped_file needs to be a valid file without extension" =
+              file.exists(paste0(ped_file, ".ped")),
+            "bed_file needs to be a valid file" =
+              (tools::file_ext(output_file) == ""),
```

Comment:
I think this should be the same as ped_file "a valid file without extension"

Reply:
i have fixed this for the four files i looked at, the only reason i didn't fix it for the rest is to avoid conflicts